geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -263,31 +263,22 @@ some `docker build`, and `docker-compose` commands.
 Main Changes
 ------------
 
-Environment variables are now stored in the env.default and env.project files.
-
-The migration process will do the following UI-related changes (visible in the html diffs):
+The migration process will changes the following things:
```

Comment:
changes -> change

Or, suggestion: "The migration process impacts the following:"